docs/pages/cheatsheet.adoc

```diff
@@ -323,47 +328,162 @@ No syntax change.
 }
 ----
 
+*Example:*
+
+[source,jsl]
+----
+constructor {
+	self.name = "";
+}
+----
+
+////
+//For later use 
+=== Transfer initializer
+
+The initializer is automatically called after the constructor is called or after the transfer object fetching. It can be used to set the values of the transient fields.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**initializer**# [purple]#**;**#
+----
+////
+
+=== Transfer create
+
+The create operation is automatically called when a new transfer object is saved for the first time. It can be used to create the mapped entity.
```

Comment:
> The create operation is automatically called when a new transfer object is saved for the first time.

Suggestion:

The create operation is called to save a new transfer object by overriding the default behaviour.

Reply:
done